site/_apps/elm-hn-pwa.txt

```diff
@@ -0,0 +1,25 @@
+---
+weight: 24 
+title: 'Elm HN PWA'
+github-title: 'skrypte/elm-hn-pwa'
+libraries: "Elm" 
+scaffolding: 'N/A'
+module-bundling: 'N/A'
+service-worker: 'partially workbox.js + application shell + data caching'
+perfomance-patterns: 'preconnect API, inline CSS, minified JS, lazy loading for JSON data'
+server-side-rendering: 'None'
+api: 'hacker-news.firebaseio.com/v0'
+hosting: 'o2switch'
+authors: 
+    - name: 'skingrapher'
+lighthouse: '91/100'
+interactive-em: '4.8s'
+interactive-faster-3g: '3.7s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=171104_BM_ce030e66965f4ea601c73316b8448f03&run=1
```

Comment:
Hmm. There's something up with this report as it doesn't appear to have data fetched until after 13 seconds in. I'll take a look at the WebPageTest report to dig into whether this was just a Lighthouse edge-case.

![image](https://user-images.githubusercontent.com/110953/33106383-a7372436-cee6-11e7-861e-0b65b85c8ab3.png)

Reply:
well, my app is bloated